tests/unit_tests/math.cpp

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2021, The Monero Project
```

Comment:
With this file we have 2 problems:
- it's a copy-paste of [contrib/epee/tests/src/misc/test_math.h](https://github.com/monero-project/monero/blob/master/contrib/epee/tests/src/misc/test_math.h), leading to duplication of the test code
- the copyright has been changed from:
`Copyright (c) 2006-2013, Andrey N. Sabelnikov, www.sabelnikov.net` 
to:
` Copyright (c) 2021, The Monero Project`.

The second point concerns the new file `src/math/math_utils.h` as well.

Reply:
> it's (almost) a copy-paste of contrib/epee/tests/src/misc/test_math.h, leading to duplication of the test code

The test in `test_math.h` was deleted, so there is no duplication.

> the copyright has been changed from:

TBH I don't know the proper way to define copyrights... let me know what this file should say and I will fix it.

Reply:
I see now.

Well the copyrights have to stay as they were, even if this file is supposed to be moved.

Reply:
Do you need to add anything to the copyright if tests are added?

Reply:
> Do you need to add anything to the copyright if tests are added?

This is a good question, that I can't answer 100%.
The best way to go around this is to add a new file with Monero's copyright, that would contain any new tests, beside the ones written by Andrey.

Reply:
What about adding a line like this:
> // Parts of this file are originally copyright (c) 2006-2013, Andrey N. Sabelnikov, www.sabelnikov.net

That pattern is used in cryptonote files.

@selsta what do you think?

Reply:
This could do, since the licences are compatible.

Reply:
Yes, add 
```
    // Parts of this file are originally copyright (c) 2006-2013, Andrey N. Sabelnikov, www.sabelnikov.net
```

to the file to be on the safe side.